xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1484,3 +1484,21 @@ bool DX::DeviceResources::SetMultithreadProtected(bool enabled) const
 
   return (wasEnabled == TRUE ? true : false);
 }
+
+bool DX::DeviceResources::IsGCNOrOlder() const
+{
+  if (CSysInfo::GetWindowsDeviceFamily() == CSysInfo::Xbox)
+    return false;
+
+  const DXGI_ADAPTER_DESC ad = GetAdapterDesc();
+
+  if (GetAdapterDesc().VendorId != PCIV_AMD)
```

Comment:
```suggestion
  if (ad.VendorId != PCIV_AMD)
```
to avoid call  GetAdapterDesc() two times